.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish IronRure
+on:
+  release:
+    types:
+      - published # Run the workflow when a new GitHub release is published
+  workflow_dispatch: # Allow manual triggering from the GitHub UI
+
+# Set permissions for the workflow.
+permissions:
+  contents: write
+  packages: write
+  checks: write          
+  pull-requests: write  
+
+jobs:
+  build:
+    uses: ./.github/workflows/ci.yml
+
+  publish:
+    if: github.event_name == 'release'
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Download Package
+        uses: actions/download-artifact@v4
+        with:
+          name: artifacts
+          path: bin/artifacts/
+
+      - name: Publish NuGet Packages to GitHub Packages
```

Comment:
Are we missing a publish to big boy Nuget?

Reply:
Added